clad/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -euxo pipefail
+source common.sh
+
+VERSION=$1
+if [[ "${VERSION}" = "trunk" ]]; then
+    VERSION=trunk-$(date +%Y%m%d)
+    BRANCH=master
+    REMOTE=heads/${BRANCH}
+else
+    BRANCH=v${VERSION}
+    REMOTE=tags/${BRANCH}
+fi
+
+URL=https://github.com/vgvassilev/clad
+CLANG_VERSION=18.1.0
```

Comment:
I would make this configurable and include it in the S3 name so we can support the plugin for multiple clang versions

Reply:
+1

Reply:
I'll take a look at that.